src/main/java/uk/ac/ebi/ega/accession/configuration/SwaggerConfiguration.java

```diff
@@ -64,6 +68,12 @@
     @Autowired
     private SwaggerApiInfoProperties swaggerApiInfoProperties;
 
+    private SecurityReference securityReference = SecurityReference.builder()
+            .reference("Authorization").scopes(new AuthorizationScope[0]).build();
+
+    private SecurityContext securityContext = SecurityContext.builder()
```

Comment:
Either do these initialization as beans or use them in the code. Avoid as much as possible complete initialization with this idiom.